specs/config/config-vocab.html

```diff
@@ -1,70 +1,171 @@
 <!DOCTYPE html>
-<html xmlns='http://www.w3.org/1999/xhtml'>
-<head>
-<meta charset='utf-8' />
-<title>OSLC Configuration Management 1.0 Part 4: RDF Vocabulary</title>
-<script src='https://raw.githack.com/oasis-tcs/tab-respec/master/builds/respec-oasis-common.js' async class='remove'></script>
-<script class='remove'>
-  var respecConfig = {
-      // specification status
-      specStatus:           "WD",
-      conformanceLabelPrefix: "CONFIG",
-
-      // the document's short name, as in http://www.w3.org/TR/short-name/
-      shortName: "oslc-config-mgmt-vocab-10",
+<html xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <meta charset="utf-8" />
+    <title>OSLC Configuration Management 1.0 Part 4: RDF Vocabulary</title>
+    <script
+      src="https://cdn.jsdelivr.net/gh/oasis-tcs/tab-respec@v2.1.19/builds/respec-oasis-common.min.js"
+      async
+      class="remove"
+    ></script>
+    <script class="remove">
+      var filePath = "/config-vocab.html";
+      var status = "WD";
+      var wdBase = "https://oslc-op.github.io/oslc-specs/specs/config";
+      var oasisBase = "https://docs.oasis-open-projects.org/oslc-op/config/v1.0";
+      var rev = "01";
+      var thisBase = wdBase;
+      if (status != "WD") {
+        thisBase = oasisBase + "/" + status.toLowerCase() + rev;
+      }
+      var respecConfig = {
+        shortName: "oslc-config-mgmt-vocab",
```

Comment:
I think you should update this value as the ReSpec snapshot will be saved under 'oslc-config-mgmt-vocab.html' AFAIK.

Reply:
Maybe this var shall be pulled up so that the filename is generated from it

Reply:
I've raised issue #414 to deal with this before publication.